src/components/EasyDropdown/index.js

```diff
@@ -53,21 +53,28 @@ const EasyDropdown = ({
   );
 
   const children = useMemo(() => {
-    return typeof initChildren === 'string' ? (
-      <Button
-        className={classNames(toggleProps.className, style.toggle)}
-        data-is-open={isOpen}
-        onClick={onToggle}
-      >
+    const childrenArray = React.Children.toArray(initChildren);
+    const hasStringChildren = childrenArray.reduce(
+      (result, child) => result || typeof child === 'string',
+      false
+    );
+
+    /*
+      If any children are raw strings, wrap all the children in a <Button />
+      Otherwise, assume there's a custom toggle component, and clone it with
+      added props for behavior.
+    */
+    return hasStringChildren ? (
+      <Button {...toggleProps} data-is-open={isOpen} onClick={onToggle}>
```

Comment:
just for my own education, what is data-is-open?

Reply:
That's an html [data attribute](https://developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Use_data_attributes), which has no effect by itself. 
I'm using it in the unit test for the component. 
Think of them as a catch-all place to put custom data. They're frequently used to give automated tests hooks to find things.